scripts/Interface/ui/CMakeLists.txt

```diff
@@ -24,6 +25,7 @@ add_custom_target(CompileUIUI DEPENDS
   CompileUIInElastic
   CompileUIReflectometer
   CompileUISANS
+  CompileUIDataProcessorInterface
```

Comment:
I would suggest not calling this dataprocessorinterface. What you have written is not meant to be used in production yet and is specific to reflectometry.

I'm also not sure about shipping this. Maybe we should keep it out of the main distirbution? @martyngigg thoughts?